website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md

```diff
@@ -157,3 +169,11 @@ Users in your Azure AD account will now be able to log into the application
 by navigating to the URL:
 
 `https://cloud.getdbt.com/enterprise-login/<login-slug>`
```

Comment:
If they're connecting to a single tenant or On-Prem instance, this URL would change the domain away from `cloud.getdbt.com`. Might be useful to flag this.

Reply:
@amychen1776 Added a new commit with this called out.

Reply:
Thank you!